bitnami/spring-cloud-dataflow/values.yaml

```diff
@@ -785,6 +790,8 @@ externalRabbitmq:
   ##
   username: guest
   password: guest
+  # vhost: /
```

Comment:
I just saw that the `externalRabbitmq` section is not present in the _values-production.yaml_ while it should appear in both files. It seems it was only implemented in the values.yaml by mistake (see https://github.com/bitnami/charts/pull/2945). Please, taking advantage of those changes, can you copy the whole
```yaml
##
## External RabbitMQ Configuration
##
## All of these values are ignored when rabbitmq.enabled is set to true
##
externalRabbitmq:
  ## Enables or disables external RabbitMQ, can be disabled when Kafka is using
  ##
  enabled: false
  ## RabbitMQ host and port
  ##
  host: localhost
  port: 5672
  ## RabbitMQ username and password, password will be saved in a kubernetes secret
  ##
  username: guest
  password: guest
  # vhost: /
  # existingPasswordSecret: name-of-existing-secret
```
code block to the _values-production.yaml_? On this way, we have unified both files and the new parameters `vhost` and `existingPasswordSecret` are present